### PR TITLE
Dockerized variant of update.sh for project dependencies.

### PR DESCRIPTION
**Pull request by jmchilton**

### CODE REVIEW

File: lib/galaxy/dependencies/pipfiles/update.sh [hunk 1]
```diff
@@ -4,21 +4,26 @@ commit=0
...
 usage() {
 cat << EOF
-Usage: ${0##*/} [-c]
+Usage: ${0##*/} [-c] [-d]
 
 Use pipenv to regenerate locked and hashed versions of Galaxy dependencies.
 Use -c to automatically commit these changes (be sure you have no staged git
-changes).
+changes). Use -d to rebuild with Pipenv from the
+galaxy/update-python-dependencies container. This container can be built be
```
Comment (mvdbeek) [inline lib/galaxy/dependencies/pipfiles/update.sh]:
```suggestion
galaxy/update-python-dependencies container. This container can be built by
```

Comment (mvdbeek) [inline lib/galaxy/dependencies/pipfiles/update.sh]:
```suggestion
galaxy/update-python-dependencies container. This container can be built by
```

**Review by nsoranzo — APPROVED** [review]
@jmchilton I've rebased, added some fixes and changed the base Docker image to `python:2.7` (since I plan to play with adding more Dockerfiles for Python 3.5, 3.6 and 3.7), hope that's OK. This image is also slightly smaller and comes with `pip` preinstalled.

**Comment by jmchilton** [comment]
Looks great to me, thanks!

**Pull request merged**